lua/grapple/settings.lua

```diff
@@ -430,6 +436,21 @@ local DEFAULT_SETTINGS = {
         title_padding = " ",
     },
 
+    --Override default floating window mappings
```

Comment:
Comment looks to have been accidentally carried over
```suggestion
```

Reply:
wdym? I see there are comments both for readme and for `settings.lua` so I'm just replicating how it is in the repo

Reply:
woops I misread this